maritime-ai-service/scripts/deploy/.env.production.template

```
@@ -196,13 +201,25 @@ LIVING_AGENT_ENABLE_AUTONOMY_GRADUATION=true
 # Target: 1000 concurrent users
 # Formula: APP_REPLICAS × GUNICORN_WORKERS = total workers
 # ─────────────────────────────────────────────────
-GUNICORN_WORKERS=8
-APP_REPLICAS=3
-APP_CPU_LIMIT=2.0
-APP_MEM_LIMIT=4G
-ASYNC_POOL_MAX_SIZE=100
+# Single-node default: 2 vCPU / 8GB RAM minimum.
+# Keep this conservative until Postgres/cache/object storage move off-host.
+GUNICORN_WORKERS=2
+APP_REPLICAS=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_REPLICAS key should go before the GUNICORN_WORKERS key

(UnorderedKey)